Dreamscapes/Ldap/Core/Ldap.php

```diff
@@ -714,7 +719,7 @@ public function startTls()
      */
     public function unbind()
     {
-        ldap_unbind($this->resource);
+        @ldap_unbind($this->resource);
```

Comment:
I am hesitant to supress PHP errors/warnings in situations where I do not call `$this->verifyOperation()` to check that the operation was successful. Are you seeing actual warnings/errors in your console while calling `Ldap::unbind()`?

I propose that either we remove the silencer or we add a call to `$this->verifyOperation()` below. Perhaps the second option would be best...